pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/PyCoverageTask.groovy

```diff
@@ -73,8 +73,10 @@ class PyCoverageTask extends PyTestTask {
         String htmlDir = streamProcessor.htmlDir
         String coverage = streamProcessor.coverage
 
-
-        FileUtils.copyDirectoryToDirectory(project.file(htmlDir), coverageOutputDir)
+        // If there is no coverage to report, then the htmlDir value will be empty
+        if ((htmlDir != null) && !htmlDir.isEmpty()) {
```

Comment:
Do we need to worry about empty directory?
It seems that it would be OK to have an empty directory copied into the output directory.
Perhaps I'm missing something in that copying behavior, but a check for `null` could be enough, couldn't it?

Reply:
You're right. I just confirmed that the fix works just fine with only a null check. I updated the commit accordingly.